docs/DeveloperGuide.md

```diff
@@ -912,27 +941,71 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
-### Adding an item
+### Finding an item
 
-1. Adding an item to StoreMando
+1. Finding an item in the inventory
 
-    1. Prerequisites: Arguments are valid and compulsory parameters are provided. No duplicate item or similar item
-       exists in the list.
+    1. Test case: `find banana`<br>
+       Expected: All items containing `banana` as a word in the name are shown.
 
-    1. Test case: `add n/Banana q/1 l/kitchen e/2020-10-10 `<br>
-       Expected: Item is added into the displayed list. Details of the added item shown in the status message.
+    1. Test case: `find */nana`<br>
+       Expected: All items containing `nana` as part of a word in the name are shown.
 
-    1. Test case: `add `<br>
-       Expected: No item is added. Error details shown in the status message.
+### Listing items
 
-    1. Other incorrect add commands to try: `add n/`, `add l/kitchen`, `...` (where compulsory fields are not
-       specified)<br>
-       Expected: Similar to previous.
+1. Listing all the items in the inventory
+
+    1. Test case: `list`<br>
+       Expected: All items are being displayed.
+
+1. Listing items in a specific location
+   
+    1. Test case: `list l/Bedroom`<br>
+       Expected: All items with `Bedroom` as the location are being displayed.
+
+1. Listing items with a specific tag 
+
+    1. Test case: `list t/food`<br>
+       Expected: All items with `food` as the tag are being displayed.
+
+### Finding items x days/weeks from expiry date
+
+1. Finding items x days from expiry date
+
+    1. Test case: `reminder 7 days`<br>
+       Expected: All the items that are 7 days from expiry are shown.
```

Comment:
Would using find be confusing for readers since we already have a find section before this?

Reply:
Also, I think it is better to state "All items that have already expired or are expiring within the next 7 days are shown".

Reply:
will do!